mk/spksrc.cross-env.mk

```diff
@@ -36,3 +36,7 @@ $(TC_VARS_MK):
 ENV += TC=$(TC)
 ENV += $(TC_ENV)
 endif
+
+ifeq ($(filter $(NCPUS),0 1),)
+COMPILE_MAKE_OPTIONS += -j$(NCPUS)
+endif
```

Comment:
Similarily, `spksrc.cross-env.mk` ends-up being included at multiple levels (e.g. `spksrc.spk.mk`, `spksrc.cross-cc.mk`, etc..) leading into possibility of getting multiple `-jX` flags in conjunction with the `MAKEFLAGS` that do follow depending of where you stand for another extra potential `-jX` flag.

Reply:
Thanks, I found  that the -j flag originally was added in the `common.mk` the `cross-env` seemed the next logical choice, but you are right it is included elsewhere. I've moved it to `compile.mk`

Reply:
While your proposal looks lean & clean i still fear your change could lead into multiple additions of `-j` flag... and getting again the `-jX job-observer forced-reset` message...

Reply:
I don't see how, the `-jN` is in the same file as before, it's even gated to only add it when `$NCPUS` is greater than 1. I've compiled 3 packages and did not see the `-jX job-observer forced-reset` message yet.